airbyte-webapp/src/views/Connection/CatalogTree/components/SyncSettingsDropdown.tsx

```diff
@@ -24,7 +24,8 @@ const Separator = styled.div`
   padding: 0 5px;
 `;
 
-const SingleValue: React.FC<IProps> = (props) => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const SingleValue: React.FC<IProps<any>> = (props) => {
```

Comment:
I think the weird component prop-drilling could be refactored out so these could be correctly typed.

Reply:
https://github.com/airbytehq/airbyte/issues/16439

Reply:
While I agree with splitting this apart if possible, is there any way we can go with `unknown` for now. The `any` here currently actually allows us to access `props.data.value` below, while `props.data` could actually be undefined (at least according to types) and then throw us an exception at runtime.

Reply:
Changed!